docs/usage/configuration-options.md

```diff
@@ -1763,6 +1763,26 @@ Example config:
 }
 ```
 
+### maxRetryAfter
+
+A remote host may return a `4xx` response with a `Retry-After` header value, which indicates that Renovate has been rate-limited.
+Renovate may try to contact the host again after waiting a certain time, that's set by the host.
+By default, Renovate tries again after the `Retry-After` header value has passed, up to a maximum of 60 seconds.
+If the `Retry-After` value is more than 60 seconds, Renovate will abort the job instead of waiting.
+
+You can configure a different maximum value using `maxRetryAfter`:
```

Comment:
The docs say that by default, Renovate will abort the run after waiting `maxRetryAfter=60`.

What happens when a user sets `maxRetryAfter=120`? Does Renovate still abort after 60 seconds?

Maybe the abort behavior depends on self-hosting/hosted app? Where the self-hosted version can stretch the waiting time a lot more, and the hosted app always aborts after 60 seconds?

Reply:
Should be abort the request, not abort the job. The job _might_ be aborted, but that's a different logic